db/migrate/20151124183140_remove_talks.rb

```diff
@@ -1,5 +1,6 @@
 class RemoveTalks < ActiveRecord::Migration
   def change
-    drop_table :talks
+    drop_table :talks do
```

Comment:
Sure thing :) I was also surprised to bump into that. :open_mouth:

Note that this empty block is only needed when `drop_table` is used within a `change` method in the migration. Within a `down` method, the simple `drop_table` statement (w/o a block) would do.

I have just added a comment in the code as to why we need an empty block, in each of the 2 relevant migrations. PR updated using `push -f`.